vj4/handler/training.py

```diff
@@ -172,7 +172,7 @@ class TrainingCreateHandler(base.Handler, TrainingMixin):
   @base.post_argument
   @base.require_csrf_token
   @base.sanitize
-  async def post(self, *, title: str, content: str, dag: str):
+  async def post(self, *, title: str, intro: str, content: str, dag: str):
```

Comment:
`intro` will be a required parameter in the future, to be displayed in the list view. Content could be optional. Based on this, I prefer the current order.